src/Uno.UI/UI/Xaml/Documents/InlineCollection.skia.cs

```diff
@@ -1094,6 +1096,8 @@ private int AdjustIndexForSurrogatePairs(int index)
 
 		internal float AverageLineHeight => _renderLines.Count > 0 ? _renderLines.Average(r => r.Height) : _lastDefaultLineHeight;
 
+		internal bool IsTrimmedByMaxLines() => _isTrimmedByMaxLines;
```

Comment:
no point making this method, this is pretty textbook side-effect free
```suggestion
		internal bool OverflowMaxLines { get; private set; }
```
and drop the field at the top